charts/oes/charts/spinnaker/config/orca-local.yml

```diff
@@ -243,7 +243,7 @@ job:
         waitForCompletion: true
         application: spin
         parameters:
-          - defaultValue: "http://oes-sapor:8085"
+          - defaultValue: "http://oes-sapor.<NAMESPACE OF OES>:8085"
```

Comment:
You don't need namespace if the custom stage job is in the same namespace. Who'll replace this before installing?

Reply:
Perhaps, we can simply keep it as old value as the default. Let's not change it

Reply:
ok Sure Abhinay